app/controllers/wiki_controller.rb

```diff
@@ -34,7 +34,14 @@ def show
         To remove the redirect, delete the tag beginning with 'redirect:'"
       end
     end
```

Comment:
Hi @r0vina !   
I guess there is this extra `end` here . Apart from this all looks perfect !!!
Great 😄 !